c/unixFileService.c

```diff
@@ -981,6 +981,43 @@ static int serveUnixFileChangeOwner (HttpService *service, HttpResponse *respons
   return 0;
 }
 
+static int serveUnixFileChangeTag (HttpService *service, HttpResponse *response) {
+  HttpRequest *request = response->request;
+  char *routeFileFrag = stringListPrint(request->parsedFile, 2, 1000, "/", 0);
+
+  if (routeFileFrag == NULL) {
+   respondWithError(response, HTTP_STATUS_BAD_REQUEST, "File name not provided");
+   return 0;
+  }
+
+  char *encodedRouteFileName = stringConcatenate(response->slh, "/", routeFileFrag);
+  char *routeFileName = cleanURLParamValue(response->slh, encodedRouteFileName);
+
+  char *codepage  = getQueryParam(response->request, "codeset");
+  char *recursive = getQueryParam(response->request, "recursive");
+  char *pattern   = getQueryParam(response->request, "pattern");
+  char *type      = getQueryParam(response->request, "type");
+
+  if (codepage == NULL || recursive == NULL || pattern == NULL || type == NULL ) {
+   respondWithError(response, HTTP_STATUS_BAD_REQUEST, "Required parameter not provided");
+   return 0;
+  }
+
+  if (!strcmp(request->method, methodPOST)) {
+    directoryChangeTagAndRespond(response, routeFileName,type, codepage, recursive, pattern);
```

Comment:
I think you formatting is a bit arbitrary. Can you please use blanks after commas.

Reply:
sure